Sources/YapDatabaseExtensions.swift

```diff
@@ -105,11 +105,11 @@ public struct YapDB {
     
     - returns: the YapDatabase instance.
     */
-    public static func testDatabase(file: String = #file, test: String = #function, operations: DatabaseOperationsBlock? = .None) -> YapDatabase {
-        let path = pathToDatabase(.CachesDirectory, name: (file as NSString).lastPathComponent, suffix: test.stringByTrimmingCharactersInSet(NSCharacterSet(charactersInString: "()")))
+    public static func testDatabase(_ file: String = #file, test: String = #function, operations: DatabaseOperationsBlock? = .none) -> YapDatabase {
+        let path = pathToDatabase(.cachesDirectory, name: (file as NSString).lastPathComponent, suffix: test.trimmingCharacters(in: CharacterSet(charactersIn: "()")))
         assert(!path.isEmpty, "Path should not be empty.")
         do {
-            try NSFileManager.defaultManager().removeItemAtPath(path)
+            try FileManager.`default`.removeItem(atPath: path)
```

Comment:
I mistakenly thought they were necessary since `default` is a keyword. I've removed them here and elsewhere. 10aa640